extensions/adobe/experience/campaign/feedbackevent.schema.json

```diff
@@ -0,0 +1,244 @@
+          "description": "Number of failed attempts before the current event"
+        },
+        "https://ns.adobe.com/experience/campaign/delivery": {
+          "title": "The campaign activity originating a message to a recipient.",
+          "type": "object",
+          "properties": {
+            "xdm:id": {
+              "title": "Delivery ID",
+              "type": "string",
+              "description": "The campaign activity originating this message."
+            },
+            "xdm:campaignId": {
+              "title": "Campaign Id",
+              "type": "string",
+              "description": "Campaign of which this delivery was a part."
+            },
+            "xdm:from": {
+              "title": "From",
+              "type": "string",
+              "description": "Address used as a from/sender/caller address. Depending on the communication channel, the `address` has a different format.\n\n  * `email`: an email address.\n  * `textMessage` or `phone`: a shortcode or phone number."
+            },
+            "xdm:type": {
+              "title": "Delivery Type",
+              "type": "string",
+              "description": "Mode of delivery for the messages sent."
+            },
+            "xdm:deliveryLabel": {
+              "title": "Delivery Label",
+              "type": "string",
+              "description": "A human-friendly name of the campaign activity which is originating this message."
+            },
+            "xdm:deliveryName": {
+              "title": "Delivery Internal Name",
+              "type": "string",
+              "description": "A human-friendly identifier of the campaign activity which is originating this message."
+            }
+          }
+        },
+        "https://ns.adobe.com/experience/campaign/message": {
+          "title": "A single message sent to a recipient.",
+          "type": "object",
+          "properties": {
+            "xdm:id": {
+              "title": "Message Identifier",
+              "type": "number",
+              "description": "Identifier of the message."
+            },
+            "xdm:subject": {
+              "title": "Message Subject",
+              "type": "string",
+              "description": "Subject line of the message."
+            },
+            "xdm:size": {
+              "title": "Size (Bytes)",
+              "type": "integer",
+              "description": "Size in bytes of the message.\n* For email, the `size` reflects the all MIME envelope of the message, including the encoded attachments if any.\n* For SMS, the `size` reflects the number of bytes necessary to deliver the message.\nIf that number is over 160 the message will be delivered over more than one SMS.",
+              "minimum": 0,
+              "maximum": 2147483647
+            },
+            "https://ns.adobe.com/experience/campaign/recipient": {
+              "title": "The recipient of this message",
+              "type": "object",
```

Comment:
Should this be an array as the same message could have multiple recipients for to/CC/BCC at the same time.

Reply:
We will be receiving different feedback events for each one of the to/CC/BCC. So, this will be one at a time.